MSAL/test/integration/native_auth/end_to_end/sign_up/MSALNativeAuthSignUpUsernameEndToEndTests.swift

```diff
@@ -270,6 +397,99 @@ final class MSALNativeAuthSignUpUsernameEndToEndTests: MSALNativeAuthEndToEndBas
         await fulfillment(of: [signUpCompleteExp])
         XCTAssertTrue(signUpVerifyCodeDelegate.onSignUpCompletedCalled)
     }
+    
+    // Use case 2.1.10 Sign up - with Email & Password, Server requires password authentication, which is not supported by the developer (aka redirect flow)
+    func test_signUpWithEmailPassword_butChallengeTypeOOB_fails() async throws {
+        guard let sut = initialisePublicClientApplication(clientIdType: .password, challengeTypes: [.OOB]) else {
+            XCTFail("Missing information")
+            return
+        }
+        
+        let username = generateSignUpRandomEmail()
+        let password = generateRandomPassword()
+        
+        let signUpFailureExp = expectation(description: "sign-up with invalid challenge type fails")
+        let signUpStartDelegate = SignUpPasswordStartDelegateSpy(expectation: signUpFailureExp)
+        
+        sut.signUp(
+            username: username,
+            password: password,
+            correlationId: correlationId,
+            delegate: signUpStartDelegate
+        )
+        
+        await fulfillment(of: [signUpFailureExp])
+        
+        // Verify error condition
+        XCTAssertTrue(signUpStartDelegate.error!.isBrowserRequired)
+    }
+    
+    // Hero Scenario 2.1.11. Sign up – Server requires password authentication, which is supported by the developer
+    // The same as 1.1.4
```

Comment:
If it is the same as the 1.1.4 why we're duplicating this test?

Reply:
Repeat confirmation on the duplication. I will remove the test case but keep the comment for future tracking.